### DISCUSSION TITLE
How to map remote cloud

### DISCUSSION THREAD

**Discussion by AP2020C** [spacedriveapp/spacedrive#1908] [Community Support]
The *only* reason I installed SpaceDrive is that I read that I can map my clouds locally with it. However when I press on Add Location it opens a window to map local folders.
How do I map the cloud?

**Comment by utkubakir** [comment]
We are working on the cloud integrations, we are still in alpha :)

**Reply by lyling** [reply]
is it ready to test in Alpah?

**Reply by utkubakir** [reply]
Not yet sadly...
